addon/components/nav-trigger-container/component.js

```diff
@@ -7,7 +7,7 @@ const {Component, inject, on} = Ember;
 export default Component.extend(RespondsToScroll, {
   layout,
   tagName: 'header',
-  classNameBindings: [':header', 'navService.navIsVisible:nav-is-visible', 'fixed:fixed', 'fixedAndScrolled'],
+  classNameBindings: [':header', 'navService.navIsVisible:nav-is-visible', 'fixed:is-fixed', 'fixedAndScrolled:is-fixed-and-scrolled'],
```

Comment:
Ember automatically takes variables in the classNameBindings and dasherizes them, so `fixedAndScrolled` would become `fixed-and-scrolled` automatically. I'd recommend updating the variables to be `isFixedAndScrolled` and `isFixed` respectively.